style.css

```diff
@@ -134,3 +147,225 @@ body {
 .social-media ul li a {
   color: #505f79;
 }
+
+.works-section {
+  display: flex;
+  flex-direction: column;
+  margin-top: 54px;
+}
+
+.card {
+  display: flex;
+  width: 327px;
+  flex-direction: column;
+  position: static;
+  border-radius: 16px;
+  margin-top: 88px;
+  padding: 16px;
+}
+
+.image1 {
+  height: 220px;
+  width: 295px;
+  border-radius: 8px;
+}
+
+.card-head {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 32px;
+  width: 236px;
+  padding-left: 0;
+}
+
+h3 {
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 44px;
+  text-align: left;
+  color: #172b4d;
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+.tonic {
+  padding-left: 0;
+  margin-top: 12px;
+}
+
+.headline1 {
+  height: 80px;
+  width: 295px;
+}
+
+.headline2 {
+  height: 124px;
+  width: 295px;
+}
+
+.canopy1 {
+  list-style: none;
+  height: 16px;
+  width: 55px;
+  left: 0;
+  font-size: 13px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 16px;
+  color: #344563;
+}
+
+.canopy {
+  font-size: 13px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.cardsub {
+  list-style: none;
+  display: flex;
+  padding-left: 0;
+  justify-content: space-between;
+  justify-self: left;
+  height: 24px;
+  width: 203px;
+}
+
+.list {
+  box-sizing: border-box;
+  padding: 4px 12px;
+  background-color: #ebebff;
+  color: #6070ff;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 16px;
+  border-radius: 8px;
+  justify-content: space-between;
+}
+
+p {
+  height: 76px;
+  width: 295px;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  color: #344563;
+}
+
+button {
+  height: 48px;
+  width: 124px;
+  box-sizing: border-box;
+  font-size: 17px;
+  color: #396df2;
```

Comment:
- [ ] [REQUIRED] Please fix the 'See Project' and 'Get My Resume' buttons color to match the original. `background: #FFFFFF;` 😊